insert_builder.go

```diff
@@ -114,3 +113,8 @@ func (b *InsertBuilder) Values(value ...interface{}) *InsertBuilder {
 	b.InsertStmt.Values(value...)
 	return b
 }
+
+func (b *InsertBuilder) IgnoreDuplicateEntries() *InsertBuilder {
```

Comment:
@lukeautry might want to call this IgnoreErrors or something similar, it will ignore more than just duplicate entry errors